test.py

```diff
@@ -234,6 +234,7 @@ async def add_test(self, shortname: str, casename: str) -> None:
 
```

Comment:
In the future, please hard wrap commitlog at 72 chars.

Reply:
Nvm, I finally understood what you meant...

Reply:
@denesb Fixed, since I had to rebase regardless, is it better now?